### PR TITLE
add selfhosted documentation

### PR DESCRIPTION
**Pull request by steven-xu-lf**
To keep things consistent, the readme file follows the same/similar format as the modified file in this merged PR -> https://github.com/Laserfiche/lf-sample-repository-api-java/pull/9
i.e. the readme file in this repo -> https://github.com/Laserfiche/lf-sample-repository-api-java

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 - .NET 6.0 core SDK
-- Git
-- Clone this repo on your local machine
+- CA, EU, or US Cloud Web Client account
 
-#### 1. Create a Service Principal
+### 1. Create a Service Principal
 
 - Log in to your account using Web Client as an administrator:
+
```
Comment (jason-chan-lf) [inline README.md]: It looks like this extra line causes extra spacing in the markdown view
image

File: README.md [hunk 2]
```diff
@@ -37,29 +37,57 @@ Sample .NET core service app that connects to a Laserfiche Cloud Repository usin
...
 
-#### 3. Create a .env file
+### 3. Clone this repo on your local machine
+
+### 4. Create a .env file
 
 - Using the app picker, go to the 'Repository Administration' page and copy the Repository ID
 - In the root directory of this project, create a .env file containing the following lines:
+```
+AUTHORIZATION_TYPE="CLOUD_ACCESS_KEY"
```
Comment (jason-chan-lf) [inline README.md]: should be CloudAccessKey

File: README.md [hunk 3]
```diff
@@ -37,29 +37,57 @@ Sample .NET core service app that connects to a Laserfiche Cloud Repository usin
...
+
+### 1. Clone this repo on your local machine
+
+### 2. Create a .env file
+
+- In the root directory of this project, create a .env file containing the following lines:
 
-AUTHORIZATION_TYPE="CloudAccessKey"
 ```
+AUTHORIZATION_TYPE="API_SERVER_USERNAME_PASSWORD"
```
Comment (jason-chan-lf) [inline README.md]: should be APIServerUsernamePassword

File: README.md (hunk 1 above)
Comment (steven-xu-lf) [reply README.md]: ok got rid of it

File: README.md (hunk 3 above)
Comment (steven-xu-lf) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (steven-xu-lf) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (jason-chan-lf) [reply README.md]: It looks like the extra whitespace is still there
https://github.com/Laserfiche/lf-sample-repository-api-dotnet-srv/blob/story/402327-add-selfhosted-documentation/README.md

Comment (steven-xu-lf) [reply README.md]: ok so after removing the other whitespace below, now the spacing matches the spacing in step 2

**Review by jason-chan-lf — APPROVED** [review]

**Pull request merged**